### PR TITLE
sending up screen width and height

### PR DESCRIPTION
**Pull request by rubinsingh**

### CODE REVIEW

File: src/6_branch.js [hunk 1]
```diff
@@ -393,7 +393,9 @@ Branch.prototype['init'] = wrap(
 					"url": utils.getWindowLocation(),
 					"user_agent": navigator.userAgent,
 					"language": navigator.language,
-					"page_has_microdata": checkForMicroData()
+					"page_has_microdata": checkForMicroData(),
+					"screen_width": window.screen.width ? window.screen.width : 0,
+					"screen_height": window.screen.height ? window.screen.height : 0
```
Comment (yntema) [inline src/6_branch.js]: - Are you sure we want to default to 0? I would confirm with whoever will be using this data
- You can do `"screen_width": screen.width || 0,`, don't need to include window

Comment (rubinsingh) [inline src/6_branch.js]: @yntema we've decided to default to -1 instead

File: src/6_branch.js [hunk 2]
```diff
@@ -393,7 +393,9 @@ Branch.prototype['init'] = wrap(
 					"url": utils.getWindowLocation(),
 					"user_agent": navigator.userAgent,
 					"language": navigator.language,
-					"page_has_microdata": checkForMicroData()
+					"page_has_microdata": checkForMicroData(),
+					"screen_width": screen.width ? screen.width : -1,
+					"screen_height": screen.height ? screen.height : -1
```
Comment (yntema) [inline src/6_branch.js]: `"screen_width": screen.width || -1,`

**Comment by yntema** [comment]
👍

**Pull request merged**